packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -51,25 +48,31 @@
                 margin-left: 0.4rem;
                 vertical-align: bottom;
             }
+
+            @include mobile {
+                display: grid;
+                position: absolute;
+                right: 1.6rem;
+            }
         }
     }
     &__stats {
         border-bottom: none;
         display: flex;
         grid-template-columns: repeat(3, 1fr) 2fr 1fr;
         grid-template-rows: 46px;
-        height: fit-content;
+        height: 50px;
         justify-content: space-between;
         margin: 2.4rem 0 2.4rem 2.9rem;
-        width: 636px;
+        width: 588px;
 
         &-cell {
             display: block;
             min-width: 65px;
-            text-align-last: center;
+            text-align: center;
 
             &-header {
-                color: $color-grey-1;
+                color: var(--text-less-prominent);
```

Comment:
By using `Text` component, we no longer need this? Right?

Reply:
Same as above.